monitoring_test.go

```diff
@@ -86,8 +86,8 @@ func TestUnmarshalMonitoringJobs(t *testing.T) {
 		t.Error("Status is not up")
 	}
 	r := j.Rules[0]
-	assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
-	assert.Equal(t, r.Value.(float64), float64(100), "RTT rule value is wrong")
+	// assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
```

Comment:
I'm a big fan of testify/assert, and I think the dependency (as it's a test dependency that doesn't get compiled into the final binary) is *absolutely worth it* for the clarity it adds to the code.

That's just my 2p however, I'm ok with you dropping it if you feel differently.